tests/test_read.py

```diff
@@ -0,0 +1,70 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import random
+import string
+from unittest import mock
+import pytest
+
+from azure.core.credentials import AzureSasCredential, AzureNamedKeyCredential
+from azure.identity import DefaultAzureCredential
+
+from azstoragetorch.io import BlobIO
+from azstoragetorch._client import AzStorageTorchBlobClient
+
+from azure.core.exceptions import ResourceExistsError
+from azure.storage.blob import BlobServiceClient
+
+
+
+@pytest.fixture
+def account_url():
+    account_name = os.environ.get('PYTORCH_STORAGE_ACCOUNT_NAME', None) or "pytorchstorageaccount"
```

Comment:
For any environment variables we expose, let's stick with the prefix of `AZSTORAGETORCH_`. So for this one, it would be `AZSTORAGETORCH_STORAGE_ACCOUNT_NAME`. The main reason that I'm suggesting this is because we own the package name `azstoragetorch`, it makes it clear that this environment variable is specific to this library and not PyTorch the library.